pages/Useful Utilities/Status-Bars.md

```diff
@@ -1,8 +1,8 @@
-# Waybar
+{{< tabs "uniqueid" >}}
```

Comment:
```suggestion
Status bars on Hyprland.

{{< tabs "uniqueid" >}}
```

Apparently hugo won't build the page if `{` is the first thing it sees. Use a different sentence if you find a better one.